### PR TITLE
feature(locksmith, unlock-app, unlock-js): Event attendee approval flow

### PR DESCRIPTION
**Pull request by julien51**
# Description

This first version of the attendee approval flow only supports individual approval/denials.
* [ ] A future flow will include bulk approval/denial
* [ ] We also need to resolve an issue with pagination (in a follow up PR)
* [ ] Refactor so we can auto-reload the Members lists

I will try to split this PR into smaller ones for easier reviews.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/__tests__/operations/keysOperations.test.ts [hunk 1]
```diff
@@ -401,5 +401,9 @@ describe('keysOperations operations', () => {
         },
       ])
     })
+
+    describe('pending keys', () => {
```
Comment (julien51) [inline locksmith/__tests__/operations/keysOperations.test.ts:405]: I need to add tests.

File: locksmith/__tests__/operations/metadataOperations.test.ts [hunk 2]
```diff
@@ -56,4 +56,7 @@ describe('metadataOperations', () => {
       })
     })
   })
+  describe('getKeyCentricData', () => {
+    it('should not fail if no tokenId is passed')
```
Comment (julien51) [inline locksmith/__tests__/operations/metadataOperations.test.ts:60]: Same here!

File: locksmith/src/controllers/v2/keyController.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { Request, Response } from 'express'
 import Normalizer from '../../utils/normalizer'
 import logger from '../../logger'
 import * as keysOperations from '../../operations/keysOperations'
+import { PAGE_SIZE } from '../../utils/constants'
```
Comment (julien51) [inline locksmith/src/controllers/v2/keyController.ts:5]: Move to the core package so we can re-use

File: packages/ui/lib/components/Form/AddressInput.tsx [hunk 4]
```diff
@@ -63,12 +63,17 @@ export const WrappedAddressInput = ({
   const [errorMessage, setErrorMessage] = useState<any>('')
   const [success, setSuccess] = useState('')
   const [address, setAddress] = useState<string>(value as string)
-
   const onReset = () => {
     setErrorMessage('')
     setSuccess('')
   }
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (searchableguy) [reply packages/ui/lib/components/Form/AddressInput.tsx:71]: I think you already pushed a separate PR for this right?

[the remark this answers is not in the document]
Comment (julien51) [reply packages/ui/lib/components/Form/AddressInput.tsx]: yeah I am rebasing..

**Pull request merged**